### PR TITLE
feat(exports): add xslx lib

### PR DESCRIPTION
**Pull request by Fabien-Verges**
refer to https://trello.com/c/ONEr8Tk8/26-3-export-des-donn%C3%A9es-lead-au-format-xlsx
Sonar : https://sonar.dev.v3.aws.lead.fm/sonar/dashboard?id=bridge-xlsx

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+8.1.0
```
Comment (tomMolin) [inline .nvmrc]: 8.7.0

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+language: node_js
+
+node_js:
+  - '8.1.0'
```
Comment (tomMolin) [inline .travis.yml]: 8.7.0
what about sonarQube instead of Travis?

File: .nvmrc (hunk 1 above)
Comment (ybonnefond) [inline .nvmrc]: 8.7.0

File: package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "bluebird": "^3.5.1"
+  },
+  "devDependencies": {
+    "codecov": "^3.0.0",
+    "eslint": "^4.9.0",
+    "exceljs": "^0.6.2",
+    "istanbul": "^0.4.5",
+    "jasmine": "^2.8.0"
+  }
+}
```
Comment (tomMolin) [inline package.json]: missing node, npm versions?

File: .travis.yml (hunk 2 above)
Comment (RomainOliva) [reply .travis.yml]: Here is the documentation for Sonar / Jenkins / Coverage integration :
- https://documentation.leadformance.com/engineering/bridge-V3/process/add-validator-integration-to-github/

File: .nvmrc (hunk 1 above)
Comment (cedgerard) [reply .nvmrc]: 👍

File: lib/errors/NotImplementedError.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+class NotImplementedError extends Error {
```
Comment (ybonnefond) [inline lib/errors/NotImplementedError.js]: KISS, if you don't use it don't keep it

File: package.json (hunk 3 above)
Comment (ThibaudAV) [reply package.json]: Added

File: package.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "bridge-xlsx",
+  "version": "1.0.0",
+  "description": "Nodejs library to manipulate xlsx files",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "NODE_ENV=test JASMINE_CONFIG_PATH=jasmine.json istanbul cover --config .istanbul.yml jasmine",
+    "cover": "open coverage/lcov-report/index.html",
+    "lint": "eslint lib test",
+    "validate": "npm run lint && npm test"
```
Comment (tomMolin) [inline package.json]: missing:
"ci-validate": "npm run validate && codecov"

File: Dockerfile.test [hunk 6]
```diff
@@ -0,0 +1,17 @@
+FROM node:8.1.0
```
Comment (tomMolin) [inline Dockerfile.test]: 8.7.0

**Pull request merged**